### ISSUE TITLE
Intel64 macOS packages will fail to install on ARM machines

### ISSUE THREAD

**Issue by DFolchA** [wazuh/wazuh-packages#2344]
|Wazuh version|Install type|Action performed|Platform|
|---|---|---|---|
| X.Y.Z-rev | Manager/API/Agent | Install/Upgrade/Remove | OS version |

Due to the restrictions on the pkgproj files the packages for 4.5.1 can only be installed in their corresponding architecture, this may cause problems for users that are already using the intel version and the WPK file. 

We should consider if we need to set this restriction as a warning for the intel package while maintaining a hard restriction for the ARM one.

This can be done by changing the `3 `in this line to a `2`.

https://github.com/wazuh/wazuh-packages/blob/f9dee4c8b74e63aedc12386854982d66673934da/macos/specs/wazuh-agent-intel64.pkgproj#L999

Related:
https://github.com/wazuh/wazuh-packages/issues/2343
https://github.com/wazuh/wazuh-packages/issues/2340

**Comment by havidarou** [comment]
Blocked because of this WPK test https://github.com/wazuh/wazuh/issues/18409

**Comment by vikman90** [comment]
Issue https://github.com/wazuh/wazuh/issues/18409 did not report critical problems. Just what we expected: the agent cannot be updated using WPKs for Intel.

Thus, I think we should remove this issue from the release.

**Comment by lchico** [comment]
I need to adjust the ETA for this task because I have been working on another task with a higher priority.

**Comment by lchico** [comment]
Blocked by issue https://github.com/wazuh/internal-devel-requests/issues/325
